dbt_subprojects/daily_spellbook/models/_sector/labels/addresses/bridge/identifiers/labels_bridges_near.sql

```diff
@@ -0,0 +1,19 @@
+{{config(
+        materialized = 'view',
+        alias = 'labels_bridges_near',
+        post_hook='{{ expose_spells(\'["near"]\',
+                                    "sector",
+                                    "labels",
+                                    \'["Sector920"]\') }}')}}
```

Comment:
```suggestion
)}}
```

Reply:
let's leave out of data explorer for now, you can still query as normal once live

Reply:
we'll keep it so that `labels.bridges` is only one which is displayed, so the intermediate tables don't add too much noise in there.